rcl/include/rcl/validate_security_context_name.h

```diff
@@ -44,8 +44,8 @@ extern "C"
 
 /// Determine if a given security context name is valid.
 /**
- * /sa The same rules as rmw_validate_namespace are used.
- * The only difference is the maximum length, which can be 255 characters.
+ * \sa The same rules as \ref rmw_validate_namespace are used.
```

Comment:
@ivanpauno I think you want to drop `\sa` here.

Reply:
See https://github.com/ros2/rcl/pull/515/commits/77aad14964b484c6091bebe0cec28169546303a9.